java/udf/src/main/java/com/risingwave/functions/PeriodDuration.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.risingwave.functions;
+
+import java.time.Duration;
+import java.time.Period;
+
+/**
+ * Combination of Period and Duration.
+ */
+public class PeriodDuration extends org.apache.arrow.vector.PeriodDuration {
```

Comment:
I wanted to re-export this class in our package so that users don't need to be aware of arrow. But it seems that Java doesn't support a way like in Rust.